lang/english.lang

```diff
@@ -312,6 +313,12 @@ several_views =                   '[views] views'
 complete_thread_marking =         'Complete thread:'
 branch_marking =                  'Current branch of thread:'
 direct_replies_marking =          'Direct replies to this posting:'
+add_vote_message_linkname =       'Like it!'
+add_vote_message_linktitle =      'Add a positive vote for this posting'
+delete_vote_message_linkname =    'Revoke vote'
+delete_vote_message_linktitle =   'Revoke vote'
+message_score1 =                  'liked by one user'
+message_score =                   'liked by [score] user'
```

Comment:
Please use plural "user**s**" (also for all other language files with the English placeholders).